conf/application.conf

```diff
@@ -140,3 +140,14 @@ mapping {
 httpHeadersWhitelist += ${mapping.nps.correlation-id}
 httpHeadersWhitelist += ${mapping.nps.request-type}
 httpHeadersWhitelist += ${mapping.nps.staff-id}
+
+# 0. API Platform ID for the QA environment
+# 1. Breathing Space Sandbox Test AppID
+api {
+  access {
+    version-1.0 {
+      whitelistedApplicationIds.0 = "ab349380-17cc-4de0-a7ac-c76baedd7133"
+      whitelistedApplicationIds.1 = "0fed2c31-b517-44a9-bd99-9c2577218a71"
```

Comment:
Shouldn't be putting real applicationId values into publicly viewable repositories